specification/notificationhubs/Notificationhubs.Management/models.tsp

```diff
@@ -0,0 +1,1600 @@
+  subscriptionId?: string;
+
+  /**
+   * Region. The value is always set to the same value as Namespace.Location, so we are deprecating
+   * this property.
+   */
+  @visibility(Lifecycle.Read)
+  region?: string;
+
+  /**
+   * Azure Insights Metrics id.
+   */
+  @visibility(Lifecycle.Read)
+  metricId?: string;
+
+  /**
+   * Time when the namespace was created.
+   */
+  @visibility(Lifecycle.Read)
+  // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
+  createdAt?: utcDateTime;
+
+  /**
+   * Time when the namespace was updated.
+   */
+  @visibility(Lifecycle.Read)
+  // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
+  updatedAt?: utcDateTime;
+
+  /**
+   * Defines values for NamespaceType.
+   */
+  namespaceType?: NamespaceType;
+
+  /**
+   * Allowed replication region
+   */
+  replicationRegion?: ReplicationRegion;
+
+  /**
+   * Namespace SKU name.
+   */
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  zoneRedundancy?: ZoneRedundancyPreference = ZoneRedundancyPreference.Disabled;
+
+  /**
+   * A collection of network authorization rules.
+   */
+  networkAcls?: NetworkAcls;
+
+  /**
+   * Collection of Notification Hub or Notification Hub Namespace PNS credentials.
+   */
+  pnsCredentials?: PnsCredentials;
+
+  /**
+   * Gets or sets endpoint you can use to perform NotificationHub
+   * operations.
+   */
+  @visibility(Lifecycle.Read)
+  serviceBusEndpoint?: string;
+
+  /**
+   * Private Endpoint Connections for namespace
+   */
+  @visibility(Lifecycle.Read)
+  privateEndpointConnections?: PrivateEndpointConnectionResource[];
+
+  /**
+   * Gets or sets scaleUnit where the namespace gets created
+   */
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  scaleUnit?: string;
+
+  /**
+   * Deprecated.
+   */
+  dataCenter?: string;
+
+  /**
+   * Type of public network access.
+   */
+  publicNetworkAccess?: PublicNetworkAccess = PublicNetworkAccess.Enabled;
+}
+
+/**
+ * A collection of network authorization rules.
+ */
+model NetworkAcls {
+  /**
+   * List of IP rules.
+   */
+  @OpenAPI.extension("x-ms-identifiers", #[])
+  ipRules?: IpRule[];
+
+  /**
+   * A default (public Internet) network authorization rule, which contains rights if no other network rule matches.
+   */
+  publicNetworkRule?: PublicInternetAuthorizationRule;
+}
+
+/**
+ * A network authorization rule that filters traffic based on IP address.
+ */
+model IpRule {
+  /**
+   * IP mask.
+   */
+  @minLength(1)
+  ipMask: string;
+
+  /**
+   * List of access rights.
+   */
+  rights: AccessRights[];
+}
+
+/**
+ * A default (public Internet) network authorization rule, which contains rights if no other network rule matches.
+ */
+model PublicInternetAuthorizationRule {
+  /**
+   * List of access rights.
+   */
+  rights: AccessRights[];
+}
+
+/**
+ * Private Endpoint Connection properties.
+ */
+model PrivateEndpointConnectionProperties {
```

Comment:
>PrivateEndpointConnectionProperties [](http://example.com/codeflow?start=6&length=35)

why not in the same typespec doc (TSP file) as PrivateEndpointConnectionResource ?